tools/web3-plugin-example/package.json

```diff
@@ -45,18 +45,18 @@
 		"prettier": "^2.7.1",
 		"ts-jest": "^28.0.7",
 		"typescript": "^4.7.4",
-		"web3": "^4.0.1",
-		"web3-core": "^4.0.1",
-		"web3-eth-abi": "^4.0.1",
-		"web3-eth-contract": "^4.0.1",
-		"web3-types": "^1.0.0",
-		"web3-utils": "^4.0.1"
+		"web3": "^4.0.3",
+		"web3-core": "^4.0.3",
+		"web3-eth-abi": "^4.0.3",
+		"web3-eth-contract": "^4.0.3",
+		"web3-types": "^1.0.2",
+		"web3-utils": "^4.0.3"
 	},
 	"peerDependencies": {
-		"web3-core": ">= 4.0.1 < 5",
-		"web3-eth-abi": ">= 4.0.1 < 5",
-		"web3-eth-contract": ">= 4.0.1 < 5",
-		"web3-types": ">= 1.0.0 < 5",
-		"web3-utils": ">= ^4.0.1 < 5"
+		"web3-core": ">= 4.0.3 < 5",
+		"web3-eth-abi": ">= 4.0.3 < 5",
+		"web3-eth-contract": ">= 4.0.3 < 5",
+		"web3-types": ">= 1.0.2 < 5",
+		"web3-utils": ">= ^4.0.3 < 5"
```

Comment:
I'm not sure if these changes are necessary

Reply:
yes, these are not necessary, just restricted deps in example to current release min patch.